pages/index.tsx

```diff
@@ -5,22 +5,31 @@ import Typography from '@mui/material/Typography'
 import Box from '@mui/material/Box'
 import Grid from '@mui/material/Grid'
 import Link from '../src/Link'
+import { getCourses }  from "../firebase/dbOperations"
 import {
 	DataGrid,
 	GridColDef,
 	GridToolbar,
 	GridRenderCellParams,
 } from '@mui/x-data-grid'
 
-interface ClassData {
-	number: string
-	aliases: string
-	department: string
-	deprecated: boolean
-	foundational: string
-	name: string
-	link: string
-	course_id: string
+interface ClassData extends Object {
+
+	aliases?: string[]
+	avgDifficulty?: number
+	avgOverall?: number
+	avgStaffSupport?: number
+	avgWorkload?: number
+	courseId?: string
+	courseNumber?: string
+	departmentId?: string
+	id?:any
+	isDeprecated?: boolean
+	isFoundational?: boolean
+	key?:any
+	name?: string
+	numReviews?: number
+	url?: string
 }
```

Comment:
Let's start consolidating the data models so we're not doing double-work / independent definition. Here, let's use the corresponding `interface` defined in `firebase/documentDataTypes.ts` (i.e., `Course`); and similarly for other client-side stuff moving forward...

Also, let's generally switch the "class" semantics over to "course" as well (filenames, variables, etc.), since that's also kind of confusing.